tf2onnx/tfonnx.py

```diff
@@ -2331,13 +2322,28 @@ def tensorflow_onnx_mapping(g, continue_on_error, custom_op_handlers):
     mapped_op = collections.Counter()
     unmapped_op = collections.Counter()
 
-    # create ops mapping for the desired opset
+    # create ops mapping for the desired opsets
     ops_mapping = {}
+
+    # load mapping for onnx domain
     for target_opset, op_map in _OPSETS:
         if target_opset <= g.opset:
             ops_mapping.update(op_map)
 
-    # apply custom ops on top of the assembled opset. We can either completment the opset
+    # load mapping for known extra opsets
+    # order matters, later mapping overrides earlier's
+    if g.extra_opset is not None:
```

Comment:
can't we use a dict("domain", mapping) so we can support any custom opsets?

Reply:
ok.
i was expecting the coming refactoring the we talked about, so i chose a simple way here for now.